File: Backjoon10818/Backjoon10818.java
package Backjoon10818;
import java.util.*;
public class Backjoon10818 {
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int max = sc.nextInt();
        int min = max;
        for (int i = 1; i < N; i++){
            int n = sc.nextInt();
            if(max < n) max = n;
            if(min > n) min = n;
        }
        sc.close();
        System.out.println(min + " " + max);
    }
}

File: Backjoon10818/Backjoon10818.c
#include <stdio.h>

int main(){
    int N, min, max;
    N = 5;
    int n[5] = {20,10,35,30,7};
    for(int i = 0; i < N; i++){
        if(i == 0){
            min = n[i];
            max = n[i];
        } 
        else{
            if(min > n[i]) 
                min = n[i];
            if(max < n[i])
                max = n[i];
        }
    }
    printf("%d %d", min, max);
    return 0;
}